profile.md

```diff
@@ -1739,6 +1739,7 @@ to the issuer's public key lifetime.
 time of the public key(s) to be used for token verification. It also equates
 to the minimum duration for a <em>new</em> key to have already been published
 before it should start to be used for signing tokens.
+A security incident may force a new key to be used sooner.
```

Comment:
This statement makes sense in the context of the next table entry, but here, I'm not really sure what it means.

Reply:
Oh.  I must have misread it.